lib/searchable_by/column.rb

```diff
@@ -3,22 +3,22 @@ class Column
     attr_reader :attr, :type, :match, :match_phrase, :wildcard
     attr_accessor :node
 
-    def initialize(attr, type: :string, match: :all, match_phrase: nil, wildcard: nil, **opts) # rubocop:disable Metrics/ParameterLists
-      if opts.key?(:min_length)
```

Comment:
we should keep this unless it's a "major" release, i.e. 0.6.x -> 0.7.x. I don't think we need a major release yet

Reply:
pls bump the version too